engine/src/main/scala/cromwell/engine/workflow/lifecycle/finalization/CopyWorkflowLogsActor.scala

```diff
@@ -8,48 +8,76 @@ import cromwell.core.Dispatcher.IoDispatcher
 import cromwell.core._
 import cromwell.core.io._
 import cromwell.core.logging.WorkflowLogger
+import cromwell.core.logging.WorkflowLogger.WorkflowLogConfiguration
 import cromwell.core.path.Path
 import cromwell.engine.workflow.WorkflowMetadataHelper
 import cromwell.filesystems.gcs.batch.GcsBatchCommandBuilder
 import cromwell.services.metadata.MetadataService.PutMetadataAction
 import cromwell.services.metadata.{MetadataEvent, MetadataKey, MetadataValue}
 
+import scala.concurrent.ExecutionContext
+import scala.util.{Failure, Success, Try}
+
 object CopyWorkflowLogsActor {
   // Commands
   case class Copy(workflowId: WorkflowId, destinationDirPath: Path)
 
-  val strategy = OneForOneStrategy(maxNrOfRetries = 3) {
+  val strategy: OneForOneStrategy = OneForOneStrategy(maxNrOfRetries = 3) {
     case _: IOException => Restart
   }
 
-  def props(serviceRegistryActor: ActorRef, ioActor: ActorRef) = Props(new CopyWorkflowLogsActor(serviceRegistryActor, ioActor)).withDispatcher(IoDispatcher)
+  def props(serviceRegistryActor: ActorRef,
+            ioActor: ActorRef,
+            workflowLogConfigurationOption: Option[WorkflowLogConfiguration] =
+            WorkflowLogger.workflowLogConfiguration,
+            copyCommandBuilder: IoCommandBuilder = GcsBatchCommandBuilder,
+            deleteCommandBuilder: IoCommandBuilder = DefaultIoCommandBuilder,
```

Comment:
Add comment:
`copy` copies between local disk off to GCS.
`delete` deletes the local disk workflow log.